core/templates/pages/contributor-dashboard-page/modal-templates/certificate-download-modal.component.ts

```diff
@@ -98,11 +104,11 @@ export class CertificateDownloadModalComponent {
     ).then((response: ContributorCertificateResponse) => {
       this.createCertificate(response);
       this.certificateDownloading = false;
-    }).catch(() => {
+    }).catch((err: HttpErrorResponse) => {
       this.errorsFound = true;
       this.certificateDownloading = false;
       this.errorMessage = (
-        'Not able to download contributor certificate');
+        err.error.error);
```

Comment:
The user might not understand the developer facing HTTP error response. Is there a particular known error we can parse?

Reply:
What I have tried is to show the errors thrown from here directly https://github.com/oppia/oppia/blob/23c6fbce44207ad636b2c970ffdce82c05086502/core/domain/suggestion_services.py#L3835C30-L3835C30